### ISSUE TITLE
Referenced entities switch.pvopt_charge_active are missing or not currently available

### ISSUE THREAD

**Issue by swests** [stevebuk1/pv_opt#200]
**Describe the bug**
Getting the following show up multiple 1000's of times in the system log

```
Logger: homeassistant.helpers.service
Source: helpers/service.py:251
First occurred: 2 April 2024 at 09:55:14 (1620 occurrences)
Last logged: 10:41:01

Referenced entities switch.pvopt_charge_active are missing or not currently available
Referenced entities switch.pvopt_discharge_active are missing or not currently available
Referenced entities switch.pvopt_hold_soc_active are missing or not currently available
```

<img width="931" alt="image" src="https://github.com/fboundy/pv_opt/assets/5365747/ac503e80-40cb-451d-a45f-66c795f4aae4">

**Additional context**
On latest version of PV_OPT.  HA 2024.3.3
Is there a way to recreate these? Or are they not needed?

**Comment by fboundy** [comment]
Noted. Should get this fixed in 3.14.0

**Comment by fboundy** [comment]
Should now be sorted in the 3.14.0. release. Let me know if it's still a problem
